.github/workflows/clang-format.yml

```diff
@@ -0,0 +1,20 @@
+name: Coding Style Check
```

Comment:
I'd still rather the filename were `coding-style-check.yml`.

I'm thinking we might want to XML lint the UI files, for example.  That could live as a separate job in the same file.